### ISSUE TITLE
migrate to jq

### ISSUE THREAD

**Issue by AladW** [AladW/aurutils#113]
https://stedolan.github.io/jq/manual/

jshon has its share of issues, such as for `-a` and `-e` (with #4 solved, running jshon for every result is noticeable with 100+ dependencies. It's also inflexible for issues like #111),  and shoehorning with `xargs` to get things in the right format.

jq is complex, really a language of its own, but should otherwise make things easier. Example from @Earnestly:

https://ptpb.pw/tbPg

**Comment by Earnestly** [comment]
It looks like jq no longer has the haskell dependency, so this makes it at least more pliable when building it from source and keeping the size down.

**Comment by AladW** [comment]
And two CVEs in the last two days:

http://www.openwall.com/lists/oss-security/2016/04/23/2
http://www.openwall.com/lists/oss-security/2016/04/24/3

I've fixed #111 by continuing on empty fields with `jshon -C`, as in your example. So the only thing jq would bring right now is easier handling of merged JSON structures (for GET and HTTP 414), but I'd rather fix the AUR RPC to have POST requests. 

Added a caveat emptor with 76afdceee70fc2f32d49de06d1264a48fe3cb747, closing this.
